### PR TITLE
PTOS recorder swap

### PR DESCRIPTION
**Pull request by mnaguit**
- from Basalt (497) to EQR120 (46621)
- for LC 20 (HN-12Z)
- firmware v2.2.0

Note:
- please double check if firmware version is correct (happy to edit if not)   ---**updated to 2.2.2**
- please double check the azimuth (change from 18 for Basalt to 198 for EQR120) ---**updated to 187**

### CODE REVIEW

File: install/streams.csv [hunk 1]
```diff
@@ -1686,7 +1686,8 @@ PTAZ,10,H,H,100,false,false,false,2022-12-08T21:30:00Z,9999-01-01T00:00:00Z
 PTHS,20,B,N,50,true,false,false,2014-08-18T23:01:00Z,2020-06-16T00:00:00Z
 PTHS,20,H,N,200,true,false,true,2002-06-21T00:00:00Z,9999-01-01T00:00:00Z
 PTOS,20,B,N,50,true,false,false,2011-06-03T04:00:00Z,2020-06-16T00:00:00Z
-PTOS,20,H,N,200,true,false,true,2011-06-03T04:00:00Z,9999-01-01T00:00:00Z
+PTOS,20,H,N,200,true,false,true,2011-06-03T04:00:00Z,2023-02-01T00:20:00Z
```
Comment (salichon) [inline install/streams.csv:1689]: llooks like it  is continuous since 2018.088 29th march 2018  
if ever @mnaguit

**Review by salichon — APPROVED** [review]

File: install/streams.csv (hunk 1 above)
Comment (mnaguit) [reply install/streams.csv:1689]: Ei @salichon , thanks.
Should we align the entry based on continuous-triggered time, or based on this recorder swap?

**Comment by mnaguit** [comment]
@salichon @CBurton90 Happy with this? 😃

**Review by CBurton90 — APPROVED** [review]

File: install/streams.csv (hunk 1 above)
Comment (salichon) [reply install/streams.csv:1689]: that s "optional" (informative in the response) 
Better to stick closer to the reality of the continuous streams in 2018 ?

**Comment by salichon** [comment]
so PTOS is the fist firmware upgrade post  canary test for 2.2.2.  - all good to me

**Review by salichon — APPROVED** [review]

**Comment by mnaguit** [comment]
Thank you Gentlemen! 👍

**Pull request merged**